src/docsfy/cli/config_cmd.py

```
@@ -0,0 +1,195 @@
+            )
+            raise typer.Exit(code=1)
+
+    # Resolve URL: explicit --host/--port wins over profile
+    if host:
+        resolved_port = port or _DEFAULT_PORT
+        # Preserve scheme from profile if available, otherwise default to https
+        profile_url = profile.get("url", "")
+        if profile_url.startswith("http://"):
+            scheme = "http"
+        else:
+            scheme = "https"
```

Ruff (0.15.6)

[warning] Use ternary operator `scheme = "http" if profile_url.startswith("http://") else "https"` instead of `if`-`else`-block

Replace `if`-`else`-block with `scheme = "http" if profile_url.startswith("http://") else "https"`

(SIM108)